Merge rsvp into admission

Timothy-Gonzalez opened the pull request:
This PR doesn't make any critical code changes - simply the structure of things changes.

- Renamed `decision` database to `admission`
- Renamed several models to better align with their real name (1to1 parity between model and type)
- Merged rsvp into admission service
  - rsvp service no longer exists
  - `/rsvp/` -> `/admission/rsvp/`
  - `GET /admission/` -> `GET /admission/not-sent`

Timothy-Gonzalez committed:
Rename decision db to admission

Timothy-Gonzalez committed:
Merge rsvp into admission

AydanPirani reviewed (approved):
lgtm! 

one question tho - why do we still have the admission entry model? @Timothy-Gonzalez  if it's not existent anywhere, can you just wipe it?

Timothy-Gonzalez committed:
Remove admission entry

No longer used

Not as sure if admission makes sense as a name anymore, we should revisit
later on. It makes more sense from an external perspective, so we'll
stick with it for now.

Timothy-Gonzalez commented:
> lgtm!
> 
> one question tho - why do we still have the admission entry model? @Timothy-Gonzalez if it's not existent anywhere, can you just wipe it?

Good question. Removed it.
